config/environments/production.rb

```diff
@@ -44,9 +44,19 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
 
-  # Use the lowest log level to ensure availability of diagnostic information
-  # when problems arise.
-  config.log_level = :debug
+  config.log_level = :info
+  config.lograge.enabled = true
+
+  config.lograge.custom_options = lambda do |event|
+    params = event.payload[:params].reject do |k|
+      ['controller', 'action'].include? k
```

Comment:
This code includes controller and action - they're not included without this. It's the same as we have in Champaign at the moment. `reject` returns a copy of the params hash, comparing them to `['controller',  'action']` allows to keep controller and action from the copy of the hash. This example is actually used in the [Lograge repository](https://github.com/roidrage/lograge), but to address your concerns - it does work as expected: 
```
Feb 24 13:01:14 ak-worker-staging production.log:  I, [2016-02-24T21:01:14.317817 #8]  INFO -- : method=POST path=/message format=html controller=message_handler action=handle status=200 duration=2574.18 view=0.16 db=2.91 params={"type"=>"update_share", "button_id"=>24, "message_handler"=>{"type"=>"update_share", "button_id"=>24}} time=2016-02-24 21:01:11 +0000
```